include/SFML/Audio/SoundFileFactory.hpp

```diff
@@ -152,8 +152,8 @@ class SFML_AUDIO_API SoundFileFactory
     // Static member data
     ////////////////////////////////////////////////////////////
     // NOLINTBEGIN(readability-identifier-naming)
-    static ReaderFactoryArray s_readers; //!< List of all registered readers
-    static WriterFactoryArray s_writers; //!< List of all registered writers
+    static inline ReaderFactoryArray s_readers; //!< List of all registered readers
```

Comment:
I don't see the point. I would rather see this removed from header and put in anonymous namespace in cpp file.

Reply:
These constants are also used in SoundFileFactory.inl so moving them to the anonymous namespace of SoundFileFactory.cpp is not a possibility.